applications/GeoMechanicsApplication/tests/shell_with_reset_displacement/README.md

```diff
@@ -0,0 +1,24 @@
+# Shell with reset displacement
+
+This tests verifies that in a shell, the prescribed displacement is not erased by the 'reset_displacement' functionality
+when that option is switched on.
+
+## Setup
+
+This test consists of a single 4-noded element (using the ShellThinElementCorotational3D4N class) of 2000mm x 2000mm . A
+schematic representation can be found in the figure below:
```

Comment:
It just needs a new line before the figure as it's showing the figure in the middle of the sentence now.
```suggestion
schematic representation can be found in the figure below:

```